### PR TITLE
Title component

### PR DESCRIPTION
**Pull request by 3lvis**
simulator screen shot - iphone x - 2018-02-04 at 12 43 57

### CODE REVIEW

File: Sources/Extensions/UIFont+Components.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ public extension UIFont {
     }
 
     public static var title2: UIFont {
-        let font = UIFont(name: "FINNTypeWebStrippet-Light", size: 28) ?? UIFont.systemFont(ofSize: 28, weight: .light)
+        let font = UIFont(name: "FINNTypeWebStrippet-Medium", size: 24) ?? UIFont.systemFont(ofSize: 24, weight: .medium)
```
Comment (ulrikus) [inline Sources/Extensions/UIFont+Components.swift]: Is this in line with the FINN-fonts from Troika?

Comment (3lvis) [reply Sources/Extensions/UIFont+Components.swift]: I don't think so, need to talk with Ingrid to decrease fonts across Finnivers first.

Comment (3lvis) [reply Sources/Extensions/UIFont+Components.swift]: Rolled back to previous values.

**Review by ulrikus — APPROVED** [review]
Not the most complex component. :)

**Pull request merged**